lib/shared/addon/components/form-gke-taints/template.hbs

```diff
@@ -0,0 +1,76 @@
+<label class="acc-label">{{t "clusterNew.googlegke.taints.label"}}</label>
+{{#if (gt taints.length 0)}}
+  <table class="table fixed no-lines small mb-10">
+    <thead>
+      <tr class="hidden-sm">
+        <th class="{{unless editing 'acc-label'}}">{{t 'clusterNew.googlegke.taints.effect.label'}}</th>
```

Comment:
You can remove the double quotes around the unless and replace the single quotes around acc-label. Also swap all singles with doubles in this file.